tests/integration/azure/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from databricks.labs.ucx.azure.resources import AzureAPIClient, AzureResources
+
+
+@pytest.fixture
+def azure_resources(ws):
```

Comment:
Remove. We don't have permissions to run integration tests that modify azure resources on github actions